### DISCUSSION TITLE
Deny log stripping

### DISCUSSION THREAD

**Discussion by JohOply** [Zrips/CMI#2152] [Feature Requests]
**Description of feature/improvement request:**
- Add an option to deny strip log with axes
- Add a permission to allow some players to be able to do it
- Add a command to toggle status (enable/disable log strip by right clicking). Be able to choose default status.
---
### Detailed explanation of what it should do, when and in what situation: ###
Prevent new vanilla behavior: right click allows to strip logs
With new option in CMI, it'll be possible to have old behaviour.
A permission and a command will allow to toggle stripping
### Basic explanation as to why this feature is needed and/or useful to have: ###
I think it would be a great addition for server owners which wants keep old behaviour or give this perm to a specific rank.
Also it could be useful to prevent grief and trolls.

Thanks for reading :)

**Comment by AllegorickBlock** [comment]
It's a good idea! I found one plugin only to do something similar, but it's not updated anymore

**Comment by mercurialmusic** [comment]
Personally I really don't want to see CMI implementing grief protections when we already have it with his other plugin, [Residence](https://www.spigotmc.org/resources/residence-1-7-10-up-to-1-14.11480/). This sort of thing is just bloat and causes confusion when the same feature exists over multiple plugins.
